src/components/ExtraPages/Help/IndicesHelp.js

```diff
@@ -1,95 +1,787 @@
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td>clt</td>
+                    <td>total cloud cover</td>
+                    <td>%</td>
+                  </tr>
+                  <tr>
+                    <td>hurs</td>
+                    <td>Near-surface relative humidity at 1.5m</td>
+                    <td>%</td>
+                  </tr>
+                  <tr>
+                    <td>huss</td>
+                    <td>Near-surface specific humidity at 1.5m</td>
+                    <td></td>
+                  </tr>
+                  <tr>
+                    <td>snw</td>
+                    <td>Amount of snow on the ground</td>
+                    <td>mm *</td>
+                  </tr>
+                  <tr>
+                    <td>psl</td>
+                    <td>Sea level pressure</td>
+                    <td>hPa</td>
+                  </tr>
+                  <tr>
+                    <td>rls</td>
+                    <td>Net surface long wave flux</td>
+                    <td>
+                      wm<sup>2</sup>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>rss</td>
+                    <td>Net surface short wave flux</td>
+                    <td>
+                      wm<sup>2</sup>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>sfcWind</td>
+                    <td>Wind speed at 10m</td>
+                    <td>
+                      ms<sup>-1</sup>
+                    </td>
+                  </tr>
+                </tbody>
+              </Table>
+            </Col>
+            <Col>
+              <Table striped size="sm">
+                <thead>
+                  <tr>
+                    <th>code</th>
+                    <th>index</th>
+                    <th>unit</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td>prsn</td>
+                    <td>Snowfall amount</td>
+                    <td>mm/day</td>
+                  </tr>
+                  <tr>
+                    <td>uas</td>
+                    <td>Eastward wind at 10m </td>
+                    <td>
+                      ms<sup>-1</sup>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>vas</td>
+                    <td>Northward wind at 10m</td>
+                    <td>
+                      ms<sup>-1</sup>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>pr</td>
+                    <td>Daily precipitation rate</td>
+                    <td>mm/day</td>
+                  </tr>
+                  <tr>
+                    <td>tas</td>
+                    <td>Mean daily near-surface air temperature</td>
+                    <td>°C</td>
+                  </tr>
```

Comment:
```suggestion
```

Reply:
@clairbarnes I copied this from https://github.com/UCL/UKCORDEX-plot-explorer-dev/pull/31#discussion_r825887254 but it feels wrong given the data we have been using up to now for the latest version of the page is "tas". Are you sure this is right?